pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1979,7 +1979,9 @@ func (s *TracerSuite) TestGetMapsTelemetry() {
 	ebpfTelemetryCollector, ok := tr.bpfErrorsCollector.(*ebpftelemetry.EBPFErrorsCollector)
 	require.True(t, ok)
 
-	mapsTelemetry := ebpfTelemetryCollector.T.GetMapsTelemetry()
+	telemetry, ok := ebpfTelemetryCollector.T.(*ebpftelemetry.EBPFTelemetry)
```

Comment:
It seems this file the only usage of `T`, `GetMapsTelemetry()`, and `GetHelpersTelemetry()` outside the package. We should explore how to move this test into the package, so we don't have to export those.

Reply:
i was planning to address that in a separate PR